pkg/RepoManages.go

```diff
@@ -372,6 +366,18 @@ func (impl RepoManagerImpl) FetchChanges(pipelineMaterialId int, from string, to
 	if err != nil {
 		return nil, err
 	}
+
+	if  pipelineMaterial.Type != sql.SOURCE_TYPE_WEBHOOK {
+		return impl.FetchGitCommitsForOtherThanWebhookType(pipelineMaterial, gitMaterial)
```

Comment:
the athor type is known so please add check for `SOURCE_TYPE_BRANCH_FIXED`   and name accordingly.

Reply:
Done